Source/AtomicApp/Player/IPCPlayerApp.cpp

```diff
@@ -206,6 +213,12 @@ namespace Atomic
 
     void IPCPlayerApp::Stop()
     {
+        if (debugPlayer_)
+        {
+            context_->GetSubsystem<JSDebugger>()->AutoReconnect = false;
+            context_->GetSubsystem<JSDebugger>()->Shutdown();
```

Comment:
ok, so shutting down explicitly it is, probably can have ```Shutdown``` turn off the auto connect